packages/vite/src/vite-node.ts

```diff
@@ -118,9 +116,17 @@ function createViteNodeApp (ctx: ViteBuildContext, invalidates: Set<string> = ne
     const node = new ViteNodeServer(viteServer, {
       deps: {
         inline: [
-          /\/node_modules\/(.*\/)?(nuxt|nuxt3|nuxt-nightly)\//,
+          /^virtual:/,
+          /\.ts$/,
+          // Rollup
+          /\0/,
+          // Common
           /^#/,
-          ...transpile({ isServer: true, isDev: ctx.nuxt.options.dev }),
```

Comment:
we can drop this as it's already set on `ssr.noExternal` here:

https://github.com/nuxt/nuxt/blob/a01c41b4d3db216aa48909aa29334c68bcff6e9a/packages/vite/src/server.ts#L67-L73